inc/notificationtargetcommonitilobject.class.php

```diff
@@ -1268,10 +1268,14 @@ function getDataForObject(CommonDBTM $item, array $options, $simple = false) {
             $tmp['##followup.isprivate##']   = Dropdown::getYesNo($followup['is_private']);
 
             // Check if the author need to be anonymized
-            if (Entity::getUsedConfig('anonymize_support_agents', $item->getField('entities_id'))
-               && ITILFollowup::getById($followup['id'])->isFromSupportAgent()
+            if (ITILFollowup::getById($followup['id'])->isFromSupportAgent()
+               && Session::getCurrentInterface() == 'helpdesk'
```

Comment:
Looks like adding `&& Session::getCurrentInterface() == 'helpdesk'` is a mistake.
```suggestion
```

Reply:
I agree it's a good solution -> 0d6685febe.